style/index.css

```diff
@@ -176,6 +176,8 @@ code {
 
 .jp-scribe-menu-content {
     padding: 5px 12px 12px 12px;
+    background-color: var(--jp-layout-color2);
```

Comment:
![Screenshot from 2019-08-17 13-16-39](https://user-images.githubusercontent.com/45380/63215205-58bddd00-c0f1-11e9-81b9-a90ed9df94d2.png)

Reply:
Note this is after a reload on binder without the previous menu bar fix... will re-screenshot from demo binder.